product_docs/docs/transporter/ba_preview/getting_started/installing/linux_x86_64/transporter_centos_7.mdx

```diff
@@ -0,0 +1,34 @@
+---
+navTitle: CentOS 7
+title: Installing EDB Transporter on CentOS 7 x86_64
+---
+
+## Prerequisites
+
+Before you begin the installation process:
+
+- Set up the EDB repository.
+
+  Setting up the repository is a one-time task. If you have already set up your repository, you don't need to perform this step.
+
+  To determine if your repository exists, enter:
+
+  `dnf repolist | grep enterprisedb`
+
+  If no output is generated, the repository isn't installed.
+
+  To set up the EDB repository:
+
+  1. Go to [EDB repositories](https://www.enterprisedb.com/repos-downloads).
+
+  1. Select the button that provides access to the EDB repo.
+
+  1. Select the platform and software that you want to download.
+
+## Install the package
+
+Install CDCReader:
```

Comment:
In a sync with Kyle, he referred to the reader as: EDB Data Migration Reader. 

Maybe we should use both names here so later references in the documentation are clear? 

```suggestion
Install the EDB Data Migration Service Reader or CDCReader:
```

We'd need to make the same edits throughout all OS variations.

Reply:
Good idea, I will change the names